syntax = "proto3";

message Account {
  int32 user_id = 1;
  string student_id = 2;
  string full_name = 3;
  string password = 4;
}

message GetAccountRequest {
  int32 user_id = 1;
  string random_string = 2;
}

message GetAccountResponse {
  Account account = 1;
}

service Accounts {
  rpc GetAccount (GetAccountRequest) returns (GetAccountResponse);
}